### ISSUE TITLE
Quasar sites don't load if cookies are blocked

### ISSUE THREAD

**Issue by prakowski** [quasarframework/quasar#1258]
### Software version

Quasar: 0.14.7
OS: All (as far as I'm aware)
Node: 
NPM:
Browsers: Chrome & Safari
iOS: 10 & 11
Android: 7.0 & 8.0
Any other software related to your bug:

### What did you get as the error?
When trying to access a quasar site like http://quasar-framework.org/quasar-play/apple/index.html while cookies are disabled on Chrome you receive the error message:

```
Failed to read the 'localStorage' property from 'Window': Access is denied for this document
```

And the page doesn't load, but remains blank.

### What were you expecting?
I was expecting the page to load.

### What steps did you take, to get the error?
I switched cookies to blocked in Chrome/Safari.

**Comment by rstoenescu** [comment]
Are you using Local Web Storage in your app?

**Comment by rstoenescu** [comment]
Blocking cookies is very unusual, but will investigate this. Thanks for reporting.

**Comment by rstoenescu** [comment]
Fixed in future v0.15.
